graft/scripts/forbidden-geth-files.txt

```diff
@@ -0,0 +1,57 @@
+# Forbidden geth upstream files that cannot be imported outside the graft directory.
```

Comment:
I considered translating both of these files to `.json` to make the loading and parsing simpler in `loadPatternFile()`.  

However, the text files are very human readable and easy to maintain. Would like any opinions about the tradeoffs here!

Reply:
No longer relevant.